### ISSUE TITLE
Log location.

### ISSUE THREAD

**Issue by LucasMW** [omar-polo/gmid#34]
Might be a dumb question, but I cannot find anywhere else.

Does gmid keeps a log of what happens and the requests it serves? Like what gemexp outputs to stdout
Where is the default location for that?
I can't find it.

Thanks in advance.

**Comment by omar-polo** [comment]
It's not a dumb question.  While it's documented, it's done a bit sparingly:

 - gmid by defaults logs to syslog, and where that actually ends up to depends on your configuration
 - if you run gmid with the `-f` (foreground) option, it will **also** print to logs to standard error (and won't put itself in the background)
 - `log access "/path/to/file"` can be used to log to a file (in addition to syslog and/or stderr)

The `log` documentation is here: https://gmid.omarpolo.com/gmid.conf.5.html#log

**Comment by LucasMW** [comment]
Thanks, I will give it a look. 
It's the first time I read about syslog, but now I have a keyword to search. 
The log option in conf might solve my problem as well.
Thanks!

**Comment by agrajag9** [comment]
An example in the docs would be nice. It took me some trial and error to figure out, but:
```
user "_gmid"
chroot "/var/gemini"
dir = "/var/gemini"
certdir = "/usr/local/etc/letsencrypt/live"
log access "gmid.log"

server "example.com" {
...
```
This will log to `/var/gemini/gmid.log`

**Comment by omar-polo** [comment]
@agrajag9 yeah, an example would be nice.  I added something in the "EXAMPLES" section of the manpage, could that have worked for you?

**Comment by agrajag9** [comment]
Yes, that's perfect! Thank you very much
